BatchDotnetTutorialFfmpeg/Program.cs

```diff
@@ -471,7 +459,7 @@ private static void MonitorTasks(BatchClient batchClient, string jobId, TimeSpan
             // Update the detail level to populate only the task id and executionInfo properties.
             detail.SelectClause = "id, executionInfo";
```

Comment:
This section should be updated to leverage ODATA predicates to filter for failed Result directly.  If the set is empty, then true==allTasksSuccessful.
Also, printing out the "message" is a bit of an anti-pattern that I would like to remove from samples.  We should demonstrate how to find the real reasons and how to plumb the depths of the collection "Details" for clarity.

This method looks like a clone of the one I've seen elsewhere in samples... so it might not have been seen and updated by Abby's work... which was supposed to include updating the instance of this method elsewhere.

Reply:
@darylmsft - This is a fair point and I'd like to follow best practices. However,  I think this is better to take as a follow-on work item, since it's not directly related (I don't think) to the refactoring for .NET Core. The quickstart/tutorial examples deliberately use small numbers of tasks, so perhaps we could simplify further and comment about using ODATA predicates for large numbers of tasks?

Reply:
@darylmsft - I updated this section to user an ODATA predicate to filter for the failed Result as you suggested. This is much cleaner than before. I didn't plumb the details for this example, however. Perhaps this could be added to the other code sample on Efficient List Queries?